### ISSUE TITLE
Supporting dialogs not cleaned up when navigating to a new page

### ISSUE THREAD

**Issue by ghost** [jtsage/jquery-mobile-datebox#164]
I have a simple page with 5 inputs which have been datebox enhanced. When I load the page, (FYI, I use Google Chrome), I see in the Elements tab in the Developer Tools that 5 hidden DIV elements of type "dialog" are created. I suspect this is to support the datebox. If I navigate away from the page (AJAX navigation enabled) and come back to the page with datebox controls, I see 5 more hidden DIVs added (in addition to the original 5). 

Basically, the datebox is adding hidden dialog elements to the page, and if you navigate back-and-forth to the page, the amount of hidden dialogs grows by N (N = # of datebox elements) each time.

Is there some way, when the parent page of a datebox control is unloaded from the DOM by JQM, these supporting hidden dialogs can get cleaned up?
